tests/purs/warning/4256.out

```diff
@@ -0,0 +1,16 @@
+Warning found:
+in module [33mMain[0m
+at tests/purs/warning/4256.purs:7:1 - 7:37 (line 7, column 1 - line 7, column 37)
+
+  No type declaration was provided for the top-level declaration of [33maddNumberSuffix'[0m.
+  It is good practice to provide type declarations as a form of documentation.
+  The inferred type of [33maddNumberSuffix'[0m was:
+  [33m                                        [0m
+  [33m  forall b6 c7 d8. b6 -> c7 -> d8 -> Int[0m
```

Comment:
Shouldn't the inferred type here include the `a` argument, which is `Int`? So something like
```diff
- forall b6 c7 d8. b6 -> c7 -> d8 -> Int
+ forall b6 c7 d8. Int -> b6 -> c7 -> d8 -> Int
```

Reply:
Ah, nevermind. The first argument has already been supplied, so this type signature is correct.